atlas-content-modeler/tutorial/models/fields/relationship/reverse_relationships.mdx

```diff
@@ -0,0 +1,40 @@
+---
+slug: /atlas-content-modeler/tutorial/models/fields/relationship/reverse_relationships
+title: Atlas Content Modeler Fields - Relationship - Reverse Relationships
+description: Learn how to use Reverse Relationships in Atlas Content Modeler.
+---
+
+**Note**: These tutorials assume that you have already [installed the plugin](../../../../introduction.mdx).
+
+## Relationship Field (Beta) - Reverse Relationships
+
+The Reverse Relationship option in Relationship Field is used to display the corresponding linked model in the publisher side of the application for that particular model.
+
+For example, if a company has employees, if you check reverse relationship in Company, it would show the linked companies in the Employee edit page.
+
+### Steps to Use the Relationship Field Reverse Relationships
+
+---
+
+1. After navigating to a model, click the + button or if it's the first field, click on the Relationship option
+
+   ![Atlas Content Modeler sync](../../../../images/field_options.png)
+
+1. Choose the Relationship Field type
+
+1. Fill in the form fields to fit your needs
+
+   ![Atlas Content Modeler sync](../../../../images/relationships_cardinality.png)
+
+1. Under Reverse Reference, check **Configure Reverse Reference** to show models that reference this model. Uncheck to hide or not show models that reference this model.
```

Comment:
> to show models that reference this model

I'm not sure this is clear enough. Can you expand a bit on this phrase/sentence so that it more clearly explains what exactly is happening? Also, it affects the graph in WPGraphQL when enabled, so worth mentioning that as well.

Reply:
A thought... in addition to the graphql bit, it not only "shows" the relationship on the reverse side, it allows management of relationships/connections from the reverse side.